Menus/BuyMenuGUI.cpp

```diff
@@ -2359,12 +2363,16 @@ void BuyMenuGUI::UpdateTotalPassengersLabel(const ACraft * pCraft, GUILabel * pL
 	if (pCraft && pCraft->GetMaxPassengers() != 0)
 	{
 		if (pCraft->GetMaxPassengers() > 0)
-			sprintf_s(buf, sizeof(buf), "%d / %d", GetTotalOrderPassengers(), pCraft->GetMaxPassengers());
+			std::snprintf(buf, sizeof(buf), "%d / %d", GetTotalOrderPassengers(), pCraft->GetMaxPassengers());
 		else 
-			sprintf_s(buf, sizeof(buf), "%d", GetTotalOrderPassengers());
+			std::snprintf(buf, sizeof(buf), "%d", GetTotalOrderPassengers());
 	}
 	else
+#if defined(__STDC_LIB_EXT1__) || defined(_WIN32)
```

Comment:
I'm seeing some inconsistency with this, you have || defined(_WIN32) here but not below.

Reply:
replaced by _WIN32 entirely since the buffer safe functions are non standard c++ and undefined in <cstring> and related headers. Also all uses are safe anyway so I don't see a problem there.